### PR TITLE
Update testing csproj to work with .NET Core

### PR DESCRIPTION
**Pull request by dbelcham**
Update of the csproj files to make them work with the TeamCity build

### CODE REVIEW

File: src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
```
Comment (andreasohlund) [inline src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj]: Don't we have to multi target here as well?

Comment (andreasohlund) [reply src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj]: I was the one doing the conversion so I just missed it. I'd say we just go ahead?

Comment (dbelcham) [reply src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj]: If it was just missed then I'll add the conversion to this PR

File: src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>PCR0001</NoWarn>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
   <ItemGroup>
 
     <ProjectReference Include="..\NServiceBus.Gateway\NServiceBus.Gateway.csproj" />
-    <Reference Include="System.Configuration" />
     <Reference Include="System.Web" />
+    <Reference Include="System.Configuration" />
```
Comment (bording) [inline src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj]: These references should be scoped to net452 only. I suppose they might just be ignored for netcoreapp2.0, but even if they are, I'd prefer to see them properly scoped.

File: src/NServiceBus.Gateway.Tests/NServiceBus.Gateway.Tests.csproj [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
+  
   <ItemGroup>
     <ProjectReference Include="..\NServiceBus.Gateway\NServiceBus.Gateway.csproj" />
-    <Reference Include="System.Configuration" />
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="System.Configuration" />
```
Comment (bording) [inline src/NServiceBus.Gateway.Tests/NServiceBus.Gateway.Tests.csproj]: Same for these references

**Pull request merged**